share/pot/mount-out.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/sh
+# shellcheck disable=SC3033,SC3040,SC3043
+:
+
+mount-out-help()
+{
+	echo "pot mount-out [-hvwr] -p pot -m mnt"
+	echo '  -h print this help'
+	echo '  -v verbose'
+	echo '  -p pot : the working pot'
+	echo '  -m mnt : the mount point inside the pot'
+}
+
+# $1 pot
+# $2 mount point
+_is_mountpoint_used()
+{
+	local _pname _mnt_p _proot
+	_pname="$1"
+	_mnt_p="${2#/}"
+	_conf=$POT_FS_ROOT/jails/$_pname/conf/fscomp.conf
+	_proot=$POT_FS_ROOT/jails/$_pname/m
+	if grep -q " $_proot/$_mnt_p$" "$_conf" ||
```

Comment:
I assume the space is intentional " ", maybe a comment would be helpful?

Reply:
Those spaces are there to enforce an exact match.
A pattern like `/mnt/test` would match both `/mnt/test` and `/mnt/test2`.
Those spaces are there to prevent `/mnt/test2` to be a match

Reply:
I understood that, but only as I was reviewing the change, hence asking for the comment.